atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -341,9 +303,13 @@ def _skip_all(self):
                                "bundles".format(component))
 
     def _get_operator_manifest(self):
-        manifests_dir = self.workflow.source.manifests_dir
-        self.log.info("Looking for operator CSV files in %s", manifests_dir)
-        operator_manifest = OperatorManifest.from_directory(manifests_dir)
+        if self.workflow.source.config.operator_manifests is None:
+            raise RuntimeError("operator_manifests configuration missing in container.yaml")
+        self.log.info("Looking for operator CSV files in %s", self.workflow.build_dir.path)
+        manifests_dir = self.workflow.source.config.operator_manifests["manifests_dir"]
+        kwargs = {'repo_dir': self.workflow.build_dir.any_build_dir.path}
+        operator_manifest = OperatorManifest.from_directory(
+            os.path.join(self.workflow.build_dir.any_build_dir.path, manifests_dir), **kwargs)
```

Comment:
why are you passing kwargs again with the same path?
originally there was just OperatorManifest.from_directory(manifests_dir)